pkg/securitypolicy/framework.rego

```diff
@@ -264,6 +264,34 @@ signal_ok(signals) {
     input.signal == signal
 }
 
+plan9_mounted(target) {
+    data.metadata.p9mounts[target]
+}
+
+default plan9_mount := {"allowed": false}
+
+plan9_mount := {"p9mounts": p9mounts, "allowed": true} {
+    not plan9_mounted(input.target)
+    some p9mount in data.policy.plan9_mounts
+    input.target == p9mount
```

Comment:
Do we actually know the exact value upfront? Here's how host adds the share: https://github.com/microsoft/hcsshim/blob/main/internal/hcsoci/resources_lcow.go#L155

For p9, the value will be https://github.com/microsoft/hcsshim/blob/main/internal/hcsoci/resources_lcow.go#L71.

So maybe some sort of regex match should happen here?

Looking at this also made me realize that we'll probably have to update the mount enforcement to support p9 as well. But that can be done in a separate PR.

Reply:
It seems like there's also this less known feature https://github.com/microsoft/hcsshim/blob/main/internal/hcsoci/resources_lcow.go#L46, which we should also take care of.

Reply:
As discussed, the scope of this PR has been changed to better reflect the enforcement needed.